src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -110,8 +112,18 @@ def run_command(self, cmd, gecko_env=True):
 
         # Run command with env
         logger.info('Running repo command', cmd=' '.join(cmd))
-        proc = subprocess.Popen(cmd, cwd=self.repo_dir)
+        proc = subprocess.Popen(
+            cmd,
+            cwd=self.repo_dir,
+            stdin=subprocess.DEVNULL,  # no interactions
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
         exit = proc.wait()
```

Comment:
There's an example in the code coverage task: https://github.com/mozilla-releng/services/blob/7063c6b/src/shipit_code_coverage/shipit_code_coverage/codecov.py#L87